### PR TITLE
375 - Bare-bones implementation of "Request Changes"

### PR DESCRIPTION
**Pull request by marcellmueller**
Implements #375

### CODE REVIEW

File: client/app/components/routes/access-requests/form/UserOperatorReview.tsx
Comment (andrea-williams) [inline client/app/components/routes/access-requests/form/UserOperatorReview.tsx:84]: technically there isn't any mention in the ticket or wireframes about Request Changes on a prime admin access request. I suggest confirming with either @nanyangpro or @hannavovk that we want to add the feature for requesting changes to prime admin requests. (Addn'l context: this PR also introduces Request Changes feature on new Operations and Operators, which we do want).

Comment (marcellmueller) [reply client/app/components/routes/access-requests/form/UserOperatorReview.tsx:84]: > technically there isn't any mention in the ticket or wireframes about Request Changes on a prime admin access request. I suggest confirming with either @nanyangpro or @hannavovk that we want to add the feature for requesting changes to prime admin requests. (Addn'l context: this PR also introduces Request Changes feature on new Operations and Operators, which we do want).

Oh damn nice catch, I think I am confused by some of this business process. The ticket did say `Pending Operator or Operation`, where do I add that for the `Operator` then if this isn't it?

Comment (andrea-williams) [reply client/app/components/routes/access-requests/form/UserOperatorReview.tsx:84]: so at the moment you're actually blocked. Brianna has a PR in progress (just waiting on some touch-ups) that separates out the approval of an Operator and a prime admin request into 2 separate review processes on the front-end.

My question for Hanna/Nan is whether we want to allow Change Requests on prime admin requests - if not, we'll want to remove that enum from the UserOperator model, and hide the button when reviewing prime admin.

Comment (marcellmueller) [reply client/app/components/routes/access-requests/form/UserOperatorReview.tsx:84]: Awesome thanks for clarifying that. Should be an easy update either way, I will bring it up at standup tomorrow.

File: client/app/components/routes/access-requests/form/UserOperatorReview.tsx
Comment (marcellmueller) [inline client/app/components/routes/access-requests/form/UserOperatorReview.tsx:124]: @andrea-williams hey just updating this after rebasing and wanted to make sure this is correct based on brief discussion at standup. We only want to be able to request changes for the operator, not the prime admin so I believe this is the logic we need?

If the operator exists already then we don't want to request changes either?

Comment (marcellmueller) [reply client/app/components/routes/access-requests/form/UserOperatorReview.tsx]: Nevermind I figured this out and passed in a prop `isPrimeAdminRequest` so we would know to hide it for that.

**Comment by andrea-williams** [comment]
@marcellmueller  I'm very sorry for the delays, I'm only just getting around to starting to review this now. Just a heads-up that one of the required checks failed (the pre-commit)

**Comment by marcellmueller** [comment]
> @marcellmueller I'm very sorry for the delays, I'm only just getting around to starting to review this now. Just a heads-up that one of the required checks failed (the pre-commit)

Oh no worries and thanks for the heads up, will fix that soon.

**Review by andrea-williams — APPROVED** [review]
Nit: could you please make the width of the "Status" column in the Operations table a bit wider? The "Changes Requested" label is getting cut off when there's no need to - we've got plenty of spare room in the table

Screenshot 2024-01-15 at 3 44 24 PM

Otherwise looking great! I've reviewed the code and run it locally to test out the behaviour - it works as expected. Nice job Marcel!

**Comment by marcellmueller** [comment]
> Nit: could you please make the width of the "Status" column in the Operations table a bit wider? The "Changes Requested" label is getting cut off when there's no need to - we've got plenty of spare room in the table

Nice suggestion, fixed thank you!

Screenshot 2024-01-15 at 3 53 35 PM

**Pull request merged**
